bot/plugins/chatrank.py

```diff
@@ -172,8 +174,11 @@ def lin_regr(x: Sequence[float], y: Sequence[float]) -> tuple[float, float]:
 
 @command('!chatplot')
 async def cmd_chatplot(config: Config, match: Match[str]) -> str:
-    user_list = optional_user_arg(match).lower().split()
-    user_list = [user.lstrip('@') for user in user_list]
```

Comment:
oh, isn't the patch just:

```diff
-    user_list = [user.lstrip('@') for user in user_list]
+    user_list = [_alias(user.lstrip('@')) for user in user_list]
```

Reply:
What happens if the provided users are the same person? I intentionally remove duplicates from the list (after expanding aliases) to ensure we don't compare ourselves. So `!chatplot kevinsjoberg kmjao` would be executed as `!chatplot kmjao`.

Reply:
maybe as another PR -- that's already existing behaviour: `!chatplot kmjao kmjao`

Reply:
Oh, didn't know that. Then, yes, it makes sense as a separate PR. Thanks for pointing it out. 🙂

Reply:
for that PR, there's a nice trick to "uniquify" a list of hashable things: `lst = list(dict.fromkeys(lst))`